app/javascript/controllers/conditional_question_controller.js

```diff
@@ -24,4 +36,18 @@ export default class extends Controller {
       })
     }
   }
+
+  displayConditionalNumeric() {
+    let value = this.element.value
+    let conditional_for = JSON.parse(this.element.dataset.info)
+
+    Object.entries(conditional_for).forEach(([key, values]) => {
```

Comment:
Maybe would be easier to read if 'key' was 'question' and 'values' would be whatever values are (answer?)